addons/mail/static/src/models/message_view.js

```diff
@@ -196,6 +196,22 @@ registerModel({
                 this.message.parentMessage
             ) ? insertAndReplace() : clear();
         },
+        /**
+         * @private
+         * @retuns {FieldCommand}
+         */
+        _computeMessageSeenIndicatorView() {
+            if (
+                this.message.isCurrentUserOrGuestAuthor &&
+                this.threadView &&
+                this.threadView.thread &&
+                this.threadView.thread.model === 'mail.channel' &&
```

Comment:
The test on model is useless because `hasSeenIndicators' is false when `thread.model !== 'mail.channel'`.